### ISSUE TITLE
Grotesquely humungous output from 0.63

### ISSUE THREAD

**Issue by DrHyde** [Ovid/Test-Differences#7]
https://rt.cpan.org/Ticket/Update.html?id=102193

**Comment by gquipster** [comment]
Hi David,

Thanks for the reply, 0.62 was a typo - I meant 0.63 :-)

In regards to RT/Github for reference the "Issues" link on Metacpan still points to RT:

https://metacpan.org/pod/Test::Differences

Gareth

**Comment by DrHyde** [comment]
Yeah, that's because the META.yml spec seems to be a moveable feast. One seems to need to occasionally change stuff at random to make it work.

**Comment by gquipster** [comment]
:-)
